src/polyfills/datepicker/datepicker.js

```diff
@@ -243,8 +243,21 @@ var componentName = "wb-date",
 		var field = calendar.field,
 			position = calendar.$field.offset();
 
-		$container
-			.attr( "style", "top:" + ( position.top + field.offsetHeight ) + "px;left:" + position.left + "px" );
+		if ( isOverlay( $container, "modal-content" ) ) {
+			$container.attr( "style", "overflow: hidden;" );
```

Comment:
This hides the overlay close button which also dismiss the actual overlay it is inside of.